js/jsmodules.go

```diff
@@ -35,7 +34,7 @@ func getInternalJSModules() map[string]interface{} {
 		"k6/experimental/redis":      redis.New(),
 		"k6/experimental/webcrypto":  webcrypto.New(),
 		"k6/experimental/websockets": &expws.RootModule{},
-		"k6/experimental/grpc":       expGrpc.New(),
+		"k6/experimental/grpc":       grpc.NewExperimental(),
```

Comment:
It kind of makes more sense to me to move the warning logic to this file only so that the grpc parts are left only about the actual implementation.

Also will be easier to remove as you need to modify only here.

Reply:
:thinking: I assumed that this would be executed even if no module imports were in the script, no? :thinking: so it is either moving this logic to the module's resolver or...

On the other side, the clean up isn't hard and I even could sketch up a PR for a long term milestone for it